content/en/tracing/setup_overview/open_standards/otel_collector_datadog_exporter.md

```diff
@@ -0,0 +1,330 @@
+
+```
+datadog:
+  api:
+    key: "<API key>"
+```
+To send the data to a different [Datadog site][6], also set the `site` parameter:
+```
+datadog:
+  api:
+    key: "<API key>"
+    site: {{< region-param key="dd_site" code="true" >}}
+```
+
+On each OpenTelemetry-instrumented application, set the resource attributes `deployment.environment`, `service.name`, and `service.version` using [the language's SDK][1]. As a fall-back, you can also configure environment, service name, and service version at the collector level for unified service tagging by following the [example configuration file][7]. The exporter attempts to get a hostname by checking the following sources in order, falling back to the next one if the current one is unavailable or invalid:
+
+1. Hostname set in the OTLP resource
+1. Manually set hostname in the exporter configuration
+1. EC2 non-default hostname (if in EC2 instance)
+1. EC2 instance id (if in EC2 instance)
+1. Fully qualified domain name
+1. Operating system host name
+
+## Ingesting OpenTelemetry traces with the collector
+
+<div class="alert alert-warning">
+The Datadog exporter for the OpenTelemetry Collector is currently in beta. It may consume high CPU and memory resources. Configuring particularly the pipeline and batch processor may take some iteration before it responds with accurate metrics given your production environment. <a href="https://docs.datadoghq.com/help/">Reach out to support</a> if it doesn't work as you expect.
+</div>
+
+The OpenTelemetry Collector is configured by adding a [pipeline][8] to your `otel-collector-configuration.yml` file. Supply the relative path to this configuration file when you start the collector by passing it in via the `--config=<path/to/configuration_file>` command line argument. For examples of supplying a configuration file, see the [environment specific setup](#environment-specific-setup) section below or the [OpenTelemetry Collector documentation][9].
+
+The exporter assumes you have a pipeline that uses the `datadog` exporter, and includes a [batch processor][10] configured with the following:
+  - A required `timeout` setting of `10s` (10 seconds). A batch representing 10 seconds of traces is a constraint of Datadog's API Intake for Trace Related Statistics.
+  <div class="alert alert-info"><strong>Important!</strong> Without this <code>timeout</code> setting, trace related metrics including <code>.hits</code>, <code>.errors</code>, and <code>.duration</code> for different services and service resources will be inaccurate over periods of time.</div>
+
+Here is an example trace pipeline configured with an `otlp` receiver, `batch` processor, `resourcedetection` processor and `datadog` exporter:
+
+```
+receivers:
+  otlp:
+    protocols:
+      grpc:
+      http:
+
+processors:
+  batch:
+    timeout: 10s
+  resourcedetection:
+    detectors: [gce, ecs, ec2, azure, system]
+
+exporters:
+  datadog/api:
+    env: prod
+    service: myservice
+    version: myversion
```

Comment:
We are going to remove these in favor of semantic conventions. This will need a more thorough docs update to document how users can set these as semantic conventions, but in the meantime I would suggest removing them

```suggestion
```